readme.md

```diff
@@ -0,0 +1,96 @@
+![](https://raw.githubusercontent.com/vercel/serve/main/media/banner.png)
+
+<div align="center">
+  <a aria-label="Vercel logo" href="https://vercel.com">
+    <img src="https://img.shields.io/badge/made%20by-vercel-%23000000">
+  </a>
+  <br>
+  <a aria-label="Install Size" href="https://packagephobia.com/result?p=serve">
+    <img src="https://packagephobia.com/badge?p=serve">
+  </a>
+  <a aria-label="Stars" href="https://github.com/vercel/serve/stargazers">
+    <img src="https://img.shields.io/github/stars/vercel/serve">
+  </a>
+  <a aria-label="Build Status" href="https://github.com/vercel/serve/actions/workflows/ci.yaml">
+    <img src="https://github.com/gamemaker1/serve/actions/workflows/ci.yaml/badge.svg">
```

Comment:
This should be `vercel` and not `gamemaker1` here, sorry I missed this before!

Reply:
Done :]